cpp/src/slice2java/Gen.h

```diff
@@ -49,7 +49,6 @@ namespace Slice
         //
         // Compose the argument lists for an operation.
         //
-        std::vector<std::string> getArgs(const OperationPtr&);
```

Comment:
Dead function. Never called.

Reply:
I find the comments rather confusing. Compose the parameter lists vs compose the argument lists??

Reply:
While unrelated, I cleaned up much this code, and fixed the doc-comments accordingly.